contrib/devtools/Makefile

```diff
@@ -41,7 +36,8 @@ mkfile_dir := $(shell cd $(shell dirname $(mkfile_path)); pwd)
 ###                                 Tools                                   ###
 ###############################################################################
 
-BIN ?= /usr/local/bin
+PREFIX ?= /usr/local
+BIN ?= $(PREFIX)/bin
 UNAME_S ?= $(shell uname -s)
 UNAME_M ?= $(shell uname -m)
 
```

Comment:
can't comment on line 47 but could you bump vuf version to 0.11.0

Reply:
do you want me to bump `BUF_VERSION ?= 0.7.0`?

Reply:
yes

Reply:
Done